ui-modules/app-inspector/app/components/entity-tree/entity-tree.html

```diff
@@ -16,7 +16,7 @@
   specific language governing permissions and limitations
   under the License.
 -->
-<entity-node ng-repeat="application in vm.applications track by application.id" entity="application" application-id="application.id"></entity-node>
+<entity-node ng-repeat="application in vm.applications | orderBy: '-startTimeUtc' track by application.id" entity="application" application-id="application.id"></entity-node>
```

Comment:
This should not be hardcoded here (always give the choice to the user)
Instead, I think there should be a button to sort by age asc/desc which would trigger the sort. The current sort should probably be remembered as well, either by storing it in a cookie or local storage

Reply:
Good idea - noted